x-pack/plugins/cases/public/containers/mock.ts

```diff
@@ -770,7 +769,16 @@ export const caseUserActionsWithRegisteredAttachmentsSnake: UserActions = [
     type: 'comment',
     action: 'create',
     id: 'create-comment-id',
-    payload: { comment: externalReferenceAttachmentSnake },
+    payload: {
```

Comment:
here as well there are some fields which are not in `userAction` type but are in `externalReferenceAttachmentSnake ` type